src/game/logic/ai/aistates.cpp

```diff
@@ -53,3 +53,12 @@ bool Out_Of_Weapon_Range_Object(State *state, void *user_data)
     return false;
 #endif
 }
+
+const Coord3D *AIStateMachine::Get_Goal_Path_Position(int i) const
+{
+    if (i < 0 || i > m_goalPath.size()) {
```

Comment:
if `i` is 0 and the size of `m_goalPath` is also 0 this check will pass and line 63 will cause a segfault

Reply:
Fixed, should have been >= not >